pose_calculator.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, detector):
         self.FOCAL_LENGTH_PIXELS = (self.SCREEN_WIDTH / 2.0) / math.tan(fov_radians / 2)
 
         # experimentally determined distance constant
-        self.DISTANCE_CONSTANT = 1.359624061
+        self.DISTANCE_CONSTANT = 1.642136009
```

Comment:
Just curious, how was this calculated?

Reply:
We ran the code on testing images and calculated an average multiplier that would match the returned value to the actual value